### PR TITLE
fix(issue): Added postMessage to handle url click in SA-UI

### PR DESCRIPTION
**Pull request by mathur07**
Jira link: https://issues.redhat.com/browse/APPAI-1389

image

### CODE REVIEW

File: src/app/stack/card-details/card-details.component.html [hunk 1]
```diff
@@ -5,9 +5,10 @@ <h2 [innerText]="details?.title"></h2>
         <p [innerText]="details?.titleDescription"></p>
     </span>
     <span class="col-sm-2" *ngIf="this.whatCard == 'security' || this.whatCard == 'compDetails'">
-        <a [href]="registrationLink" title="Sign up with Snyk" target="_blank">
+        <a [href]="registrationLink" title="Sign up with Snyk" target="_blank"
+            (click)="this.handleUrl.open($event,registrationLink)">
```
Comment (arajkumar) [inline src/app/stack/card-details/card-details.component.html:9]:
```suggestion
            (click)="this.handleUrl.open($event, registrationLink)">
```

Comment (mathur07) [reply src/app/stack/card-details/card-details.component.html:9]: Ya it is backward compatible, cmd + click behaviour will continue to exist.

**Comment by arajkumar** [comment]
@mathur07 Can you answer my question?

**Comment by mathur07** [comment]
@arajkumar Yes it is backward compatible, cmd + click behaviour will continue to exist.
https://github.com/fabric8-analytics/fabric8-analytics-stack-report-ui/pull/155#discussion_r459974712

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**